### ISSUE TITLE
About the glyph: `

### ISSUE THREAD

**Issue by Photonico** [tonsky/FiraCode#1487]
About the glyph `

I have closed ligature. I find that if it is entered alone, the font will choose the glyph ` U+0060“grave” `;
But if I continuously enter two or more, the font will choose another glyph that I don't know, perhaps ` U+0060“grave.case” `
I have customized the glyph  ` U+0060“grave” `; 
How to let the font still choose glyph  ` U+0060“grave”  when I enter two or more ` ?

**Comment by tonsky** [comment]
I don’t think any other glyph is selected. Please check again

**Comment by Photonico** [comment]
> I don’t think any other glyph is selected. Please check again

Hi,

I have checked again, only glyph ` is modified in Fira Code source file.
![image](https://user-images.githubusercontent.com/17745311/202467948-e14ba35f-08f5-4f92-85a5-c82c77e55478.png)

You can see the ` are different between Line5 and Line6.

**Comment by lihaoyu1234** [comment]
Yes, it's the same on my computer, and the font Inconsolata also has this problem. I don't know how to turn off this function.

**Comment by hugh-lx** [comment]
The same to caron(ˇ), acute(´), breve(˘), etc.
It seems to be related to the 'ccmp' feature.
I turn off this feature and these substitutions don't happen now.
But I don't know if turning off the feature will cause any other problem.
